src/viam/ur/module/ur_arm.cpp

```diff
@@ -51,14 +52,24 @@ std::chrono::milliseconds unix_now_ms() {
     return chrono::duration_cast<chrono::milliseconds>(chrono::system_clock::now().time_since_epoch());
 }
 
+template <typename T>
+[[nodiscard]] constexpr decltype(auto) degrees_to_radians(T&& degrees) {
```

Comment:
These are more complicated than you might expect because I want `T` to be able to be an Eigen expression template type, and to return an expression template representing the deferred conversion. Need to double check that `decltype(auto)` is right here, but I think it is.